core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts

```diff
@@ -197,7 +197,7 @@ export class EnvAssetsSectionComponent implements OnInit {
     this.subscription = this.http.get(this.relativeUrl, payload, this.service.id).subscribe(
       (response) => {
 
-        if((response.data == undefined) || (response.data.length == 0)){
+        if((response.data == undefined) || (response.data.length == 0) || (response.data.count == 0)){
```

Comment:
You can remove (response.data.length == 0) condition.

Reply:
Removed